packages/strapi-admin/controllers/Admin.js

```diff
@@ -40,5 +41,23 @@ module.exports = {
       // Fallback, render admin page
       ctx.body = strapi.admin.services.admin.generateAdminIndexFile();
     }
+  },
+
+  uninstallPlugin: async ctx => {
+    try {
+      const { plugin } = ctx.params;
+
+      strapi.reload.isWatching = false;
+
+      strapi.log.info(`Uninstalling ${plugin}...`);
+      exec(`strapi uninstall ${plugin}`);
```

Comment:
Take a look at this https://github.com/strapi/strapi/commit/8a8d85356afe704b1a4a154c49216b6348516e73#diff-14d02270f67a6dcfa099642fbe9fff85R91 to ensure it's harmonized